### PR TITLE
Modify shopping cart invoice email to meet CME requirements.

### PR DESCRIPTION
**Pull request by dcadams**

### CODE REVIEW

**Comment by dcadams** [comment]
@stvstnfrd

File: lms/djangoapps/shoppingcart/models.py [hunk 1]
```diff
@@ -375,6 +375,8 @@ def send_confirmation_emails(self, orderitems, is_order_type_business, csv_file,
             )
             # Send a unique email for each recipient. Don't put all email addresses in a single email.
             for recipient in recipient_list:
+                # Some of the names in the db end in white space.
+                name = self.user.profile.name.strip()
```
Comment (stvstnfrd) [inline lms/djangoapps/shoppingcart/models.py]: Can we call this `user_name`?

File: lms/envs/aws.py [hunk 2]
```diff
@@ -209,6 +209,8 @@
 CONTACT_EMAIL = ENV_TOKENS.get('CONTACT_EMAIL', CONTACT_EMAIL)
 BUGS_EMAIL = ENV_TOKENS.get('BUGS_EMAIL', BUGS_EMAIL)
 PAYMENT_SUPPORT_EMAIL = ENV_TOKENS.get('PAYMENT_SUPPORT_EMAIL', PAYMENT_SUPPORT_EMAIL)
+PAYMENT_SUPPORT_PHONE = ENV_TOKENS.get('PAYMENT_SUPPORT_PHONE', PAYMENT_SUPPORT_PHONE)
```
Comment (stvstnfrd) [inline lms/envs/aws.py]: Stanford-specific settings go in our subdirectory: `openedx/stanford/lms/envs/aws.py`

File: openedx/stanford/lms/envs/common.py [hunk 3]
```diff
@@ -76,6 +76,8 @@
 }
 ORA2_RESPONSES_DOWNLOAD_ROUTING_KEY = HIGH_MEM_QUEUE
 PAYMENT_CONFIRM_EMAIL = PAYMENT_SUPPORT_EMAIL
+PAYMENT_SUPPORT_PHONE = '1112223344'
+PAYMENT_PLATFORM_NAME = 'Payment Platform Name'
```
Comment (stvstnfrd) [inline openedx/stanford/lms/envs/common.py]: Please alphabetize.

File: lms/templates/emails/order_confirmation_email.txt [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
+${_("If you have any questions, please contact us by email or phone {payment_support_phone}").format(payment_support_phone=payment_support_phone)}
 % endif
 
 ${_("Thank you,")}
 
 % if payment_email_signature:
 ${payment_email_signature}
 % else:
-${_("The {platform_name} Team").format(platform_name=platform_name)}
+${_("Your {payment_platform_name} Team {billing_email}").format(payment_platform_name=payment_platform_name, billing_email=payment_support_email)}
```
Comment (stvstnfrd) [inline lms/templates/emails/order_confirmation_email.txt]: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines

File: lms/templates/emails/order_confirmation_email.txt [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
+${_("Hi {name},").format(name=name)}
 
 ${_("Your payment was successful. You will see the charge below on your next credit or debit card statement under the company name {merchant_name}.").format(merchant_name=settings.CC_MERCHANT_NAME)}
 
+${_("You can access your course from your online learning dashboard at: {dashboard_url}").format(dashboard_url=dashboard_url)}
 % if marketing_link('FAQ'):
 ${_("If you have billing questions, please read the FAQ ({faq_url}) or contact {billing_email}.").format(billing_email=payment_support_email, faq_url=marketing_link('FAQ'))}
 % else:
-${_("If you have billing questions, please contact {billing_email}.").format(billing_email=payment_support_email)}
+${_("If you have any questions, please contact us by email or phone {payment_support_phone}").format(payment_support_phone=payment_support_phone)}
```
Comment (stvstnfrd) [inline lms/templates/emails/order_confirmation_email.txt]: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines

File: lms/envs/aws.py [hunk 6]
```diff
@@ -209,6 +209,7 @@
 CONTACT_EMAIL = ENV_TOKENS.get('CONTACT_EMAIL', CONTACT_EMAIL)
 BUGS_EMAIL = ENV_TOKENS.get('BUGS_EMAIL', BUGS_EMAIL)
 PAYMENT_SUPPORT_EMAIL = ENV_TOKENS.get('PAYMENT_SUPPORT_EMAIL', PAYMENT_SUPPORT_EMAIL)
+PAYMENT_PLATFORM_NAME = ENV_TOKENS.get('PAYMENT_PLATFORM_NAME', PAYMENT_PLATFORM_NAME)
```
Comment (stvstnfrd) [inline lms/envs/aws.py]: Should this also go in `openedx/stanford/lms/envs/aws.py`?

**Comment by caesar2164** [comment]
awesome, no silly defaults! 👍

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**